sgpc3_with_shtc1/sgpc3_with_shtc1_example_usage.c

```diff
@@ -31,12 +31,8 @@
 
 #include "sgpc3_with_shtc1.h"
 
-/* TO USE CONSOLE OUTPUT (printf) AND WAIT (sleep) PLEASE ADAPT THEM TO YOUR
- * PLATFORM.
- *
- * #include <stdio.h> // printf
- * #include <unistd.h> // sleep
- */
+#include <stdio.h>   // printf
+#include <unistd.h>  // sleep
```

Comment:
I think the `unistd.h` include isn't really needed.

Also can you add the following below the includes:
```

/* TO USE CONSOLE OUTPUT (printf) YOU MAY NEED TO ADAPT THE
 * INCLUDE ABOVE OR DEFINE IT ACCORDING TO YOUR PLATFORM.
 * #define printf(...)
 */
```

Reply:
Done.